src/utils.js

```diff
@@ -1,5 +1,5 @@
 export function numberIsFinite(value) {
-  return typeof value === 'number' && isFinite(value);
+  return typeof value === 'number' && Number.isFinite(value);
```

Comment:
This was done intentionally to not break IE11 (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isFinite#Browser_compatibility)

Reply:
Fixed in fcc9189